packages/wrangler/src/index.tsx

```diff
@@ -421,7 +428,11 @@ export async function main(argv: string[]): Promise<void> {
     "👂 Start a local server for developing your worker",
     (yargs) => {
       return yargs
-        .positional("filename", { describe: "entry point", type: "string" })
+        .positional("filename", {
+          describe: "entry point",
+          type: "string",
+          demandOption: true,
```

Comment:
using `<filename>` (i.e the pointy brackets) implies `demandOption:true`, but ok.

Reply:
That is true for "runtime" but for type checking the typings are only able to indicate that `args.filename` must not be undefined if the `demandOption` property is true.

Reply:
Didn't we recently add the PR from James that allowed for empty `filename` to attempt to run on a root `index`? This would override that. 

https://github.com/cloudflare/wrangler2/pull/79

Reply:
The PR you link to was replaced by https://github.com/cloudflare/wrangler2/pull/196/files
Even in this new PR the `filename` option is still required, because that is passed to esbuild as its entry point.

Reply:
Hmmm, James had mentioned it defaulting to `index` the 196 PR will be merged soon and we can check the behavior with this change too